ads-on-other-themes.rst

```diff
@@ -0,0 +1,127 @@
+* Projects that opt out of normal advertising, as previously supported, will now be shown *only* ads from our Community Ads partners.
+* Users will be able to opt out of all paid ads with a profile setting, but will still see Community Ads.
+
+Community Ads
+~~~~~~~~~~~~~
+
+There are a large number of projects that we care about in the software ecosystem. A large number of them are operated like we have for the past 6 years, with almost no income. Our new Community Ads program will highlight one of these projects each month.
+
+We'll show 10% of our ad inventory each month to support an open source project that we care about.
+We are working to set up an application process to be considered,
+but we've already picked our first Community Ad recipients: **Sustain** and **Beeware**.
+Beeware is running their own sustainability campaign,
+Sustain is about sustainability,
+and we're hoping to help out!
+
+`Beeware <http://pybee.org/>`_ is a collection of projects that can be used to help develop, debug and launch Python software. Each tool follows the Unix philosophy of doing one thing well. The primary goal is to bring Python to the browser, mobile devices, and everywhere else that we wish Python ran natively. 
+
+`Sustain <https://sustainoss.org/>`_ is a one-day conference about open source sustainability. We ran ads for it earlier this month, and it is now sold out.
+
+Project & User opt out changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Previously we've allowed project owners to opt out of advertising. We're now expanding this to allow users to opt out of advertising as well.
+However,
+we're using the opportunity of launching our Community Ads to be able show them to folks who out opt.
+We know some folks disagree with the ideas of companies paying for their attention,
+but we believe that seeing community projects promoted is something that should have value for people.
+
+To repeat,
+**Users and Projects can now opt out of paid ads, but will be shown Community Ads instead**.
+
+What we're doing with the money
+-------------------------------
+
+We have a couple primary things that we are spending money on,
+and plan to spend money on:
+
+* Paying full-time salaries for the primary maintainers of Read the Docs (Eric Holscher & Anthony Johnson)
+* Paying for contractors in order to support our users
+
+I'd like to talk a bit more each of these.
+
+Paying Salaries
+~~~~~~~~~~~~~~~
+
+Read the Docs is a massive internet service.
+We serve around 30 million _pageviews_ a month (calculated by Google Analytics, this is over 200 million _requests_ a month.)
+We are currently running the entire show with two full-time staff,
+which is incredibly understaffed compared to most internet services.
+
+We hope that over time we will be able to pay salaries for more people,
+but even if we have another $56,000 a year,
```

Comment:
Where does 56k come from? If this isn't referenced anywhere, probably best to leave it out.